renderdoc/driver/gl/gl_common.h

```diff
@@ -322,6 +323,7 @@ extern bool IsGLES;
   EXT_TO_CHECK(43, 31, ARB_program_interface_query)              \
   EXT_TO_CHECK(43, 31, ARB_shader_storage_buffer_object)         \
   EXT_TO_CHECK(43, 31, ARB_stencil_texturing)                    \
+  EXT_TO_CHECK(31, 99, ARB_texture_buffer_object)                \
```

Comment:
Can you add this at the start please? The extensions are sorted by the core version number currently.